onnxruntime/test/testdata/transform/propagate_cast/gen_propagate_cast.py

```diff
@@ -128,7 +129,7 @@ def gen_fuse_sibling_casts(model_path):
              type_to_string(type2), nodes, inputs, outputs, [])
 
 
-def flip_type(flip, type):
+def flip_type(type, flip=True):
```

Comment:
nit: use 'enabled' instead of 'flip'

Reply:
I think I like 'flip' better than 'enable' because it is more readable and carries the meaning better.